### PR TITLE
feat(account): introduce execute method

### PR DESCRIPTION
**Pull request by janek26**
This PR aims to support batch transactions natively. 

To archive this it's necessary to introduce a new method and not further inherit the interface of `invokeFunction` which is designed to always match the sequencers api, which in any case accepts one call only.

### CODE REVIEW

**Comment by dhruvkelawala** [comment]
Looks good to me

File: src/account/default.ts [hunk 1]
```diff
@@ -36,41 +41,52 @@ export class Account extends Provider implements AccountInterface {
    * @param transaction - transaction to be invoked
    * @returns a confirmation of invoking a function on the starknet contract
    */
-  public override async invokeFunction(
-    { contractAddress, calldata = [], nonce, entrypoint, ...invocation }: Invocation,
-    _abi?: Abi
+  public async execute(
```
Comment (0xs34n) [inline src/account/default.ts:44]: Might want to change the comment on this function above `Invoke a function` -> `call execute function in account contract` also the reference is probably not relevant for this function

File: src/account/interface.ts [hunk 2]
```diff
@@ -39,9 +40,10 @@ export abstract class AccountInterface extends ProviderInterface {
    *
    * @returns response from addTransaction
    */
-  public abstract override invokeFunction(
-    invocation: Invocation,
-    abi?: Abi
+  public abstract execute(
```
Comment (0xs34n) [inline src/account/interface.ts:43]: same comment as the implementation, change comment from Invoke -> call execute on account contract

**Review by 0xs34n — CHANGES_REQUESTED** [review]
Looks good! Small changes on the comments on the functions.

Also, I noticed some formatting changes that I think are being auto-changed from our editors. Not sure what's causing this as we have the prettier config file

**Comment by janek26** [comment]
@0xs34n thanks! done!

**Pull request merged**